vulns/CVE-2024-5321.json

```diff
@@ -0,0 +1,62 @@
+{
+  "id": "CVE-2024-5321",
+  "modified": "2024-07-17T13:06:48Z",
+  "published": "2024-07-17T13:06:48Z",
+  "summary": "Incorrect permissions on Windows containers logs",
+  "details": "A security issue was discovered in Kubernetes clusters with Windows nodes where BUILTIN\\Users may be able to read container logs and NT AUTHORITY\\Authenticated Users may be able to modify container logs.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/kubelet"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:L/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "type": "SEMVER",
+          "events": [
+            {
+              "introduced": "1.27.0"
+            },
+            {
+              "last_affected": "1.27.15"
```

Comment:
It's already fixed in 1.27.16. Will update it.

Reply:
done
https://github.com/kubernetes-sigs/cve-feed-osv/pull/7/commits/332a8cda2071aa355600a114cd14d6bae80f3b0f